RUFAS/biophysical/animal/animal_growth/animal_growth.py

```diff
@@ -1,2 +1,412 @@
+
+    @staticmethod
+    def _calculate_cow_target_daily_growth(
+        reproduction_properties: ReproductionProperties, general_properties: GeneralProperties
+    ) -> float:
+        """
+        Calculates the target daily growth for cows.
+
+        Parameters
+        ----------
+        reproduction_properties: ReproductionProperties
+            Animal properties that are related to animal reproduction.
+        general_properties: GeneralProperties
+            Animal properties that are general or are used to determine many animal outcomes.
+
+        Returns
+        -------
+        float
+            The daily growth rate for cows (kg).
+        """
+        if reproduction_properties.calves == 1:
+            if general_properties.days_in_preg < 1:
+                target_adg_cow = (
+                    (0.92 - 0.82)
+                    * 0.96
+                    * general_properties.mature_body_weight
+                    / reproduction_properties.calving_interval
+                )
+            else:
+                target_adg_cow = (0.92 * general_properties.mature_body_weight - general_properties.body_weight) / (
+                    reproduction_properties.gestation_length - general_properties.days_in_preg + 1
+                )
+        elif reproduction_properties.calves == 2:
+            if general_properties.days_in_preg < 1:
+                target_adg_cow = (
+                    (1 - 0.92) * 0.96 * general_properties.mature_body_weight / reproduction_properties.calving_interval
+                )
+            else:
+                target_adg_cow = (general_properties.mature_body_weight - general_properties.body_weight) / (
+                    reproduction_properties.gestation_length - general_properties.days_in_preg + 1
+                )
+        else:
+            target_adg_cow = 0
+        return target_adg_cow
+
+    @staticmethod
+    def _calculate_cow_body_weight_tissue_change(
+        animal_growth_properties: AnimalGrowthProperties,
+        reproduction_properties: ReproductionProperties,
+        general_properties: GeneralProperties,
+    ) -> tuple[float, float]:
+        """
+        Calculates the body weight tissue growth for cows.
+
+        Parameters
+        ----------
+        animal_growth_properties: AnimalGrowthProperties
+            Animal properties that are related to body weight growth.
+        reproduction_properties: ReproductionProperties
+            Animal properties that are related to animal reproduction.
+        general_properties: GeneralProperties
+            Animal properties that are general or are used to determine many animal outcomes.
+
+        Returns
+        -------
+        tuple[float, float]
+            The body weight tissue growth for cows (kg), and the updated tissue changed (kg).
+        """
+        updated_tissue_changed = animal_growth_properties.tissue_changed
+        if not general_properties.days_in_milk == 0:
```

Comment:
```suggestion
        if general_properties.is_milking:
```
This should be functionally equivalent, because days_in_milk should never be negative.